src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorTest.php

```diff
@@ -123,19 +123,6 @@ public function testGetValueReturnsNullIfPropertyNotFoundAndExceptionIsDisabled(
         $this->assertNull($this->propertyAccessor->getValue($objectOrArray, $path), $path);
     }
 
-    /**
-     * @group legacy
-     * @dataProvider getPathsWithMissingProperty
-     */
-    public function testGetValueReturnsNullIfPropertyNotFoundAndExceptionIsDisabledUsingBooleanArgument($objectOrArray, $path)
-    {
-        $this->expectDeprecation('Since symfony/property-access 5.3: Passing a boolean as the fourth argument to "Symfony\Component\PropertyAccess\PropertyAccessor::__construct()" is deprecated. Pass a combination of bitwise flags as the second argument instead (i.e an integer).');
```

Comment:
Please also remove `ExpectDeprecationTrait` from this test:

https://github.com/symfony/symfony/blob/2c7ba63fe324bbe4206fb92f0c75b238ad4d52ce/src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorTest.php#L15

see https://github.com/symfony/symfony/pull/41298#discussion_r635621091